js/templates/modals/connectionManagement/configurationForm.html

```diff
@@ -22,94 +20,180 @@ <h2 class="h3 clrT txUnl txCtr"><%= ob.title %></h2>
   </div>
   <% } %>  
 
-  <form class="padMdKids padStack">
-  <% if (!ob.default) { %>
-    <div class="flexRow">
-      <div class="col3">
-        <label for="serverConfigName" class="required"><%= ob.polyT('connectionManagement.configurationForm.name') %></label>
-      </div>
-      <div class="col9">
-      <% if (ob.errors.name) print(ob.formErrorTmpl({ errors: ob.errors.name })) %>
-        <input type="text" class="clrBr clrSh2 js-inputName" name="name" id="serverConfigName" value="<%= ob.name %>" placeholder="<%= ob.polyT('connectionManagement.configurationForm.placeholderName') %>" >
-      </div>
-    </div>
-    <div class="flexRow">
-      <div class="col3">
-        <label for="serverConfigServerIp" class="required"><%= ob.polyT('connectionManagement.configurationForm.serverIp') %></label>
-      </div>
-      <div class="col9">
-      <% if (ob.errors.serverIp) print(ob.formErrorTmpl({ errors: ob.errors.serverIp })) %>
-        <input type="text" class="clrBr clrSh2" name="serverIp" id="serverConfigServerIp" value="<%= ob.serverIp %>"placeholder="<%= ob.polyT('connectionManagement.configurationForm.placeholderServerIp') %>">
+  <form class="padMdKids padStack <% if (ob.useTor) print('useTor') %>">
+    <% if (ob.isNew) { %>
+      <div class="flexRow">
+        <div class="col3">
+          <label class="required"><%= ob.polyT('connectionManagement.configurationForm.lblServerType') %></label>
+          <div class="clrT2 tx6"><%= ob.polyT('connectionManagement.configurationForm.helperServerType') %></div>
+        </div>
+        <div class="col9">
+          <div class="btnStrip">
+            <div class="btnRadio">
+              <input type="radio"
+                     name="serverType"
+                     value="BUILT_IN"
+                     id="serverConfigBuiltin"
+                     data-field-builtin
+                     data-field-standalone
+                     <% if (ob.builtIn === true) { %>checked<% } %>>
+              <label class="gutterHSm" for="serverConfigBuiltin"><span><%= ob.polyT('connectionManagement.configurationForm.serverTypeBuiltin') %></span><span class="toolTip" data-tip=""><i class="ion-help-circled"></i></span></label>
```

Comment:
Why is the data-tip value blank? No tooltip appears when the help icon is hovered over.

Reply:
just waiting on copy here. Whoever wants is welcome to provide it.

Reply:
Built-in: "This server is created and controlled by this app on this computer, and only runs when this app is running. This is recommended for most users."

Stand-Alone: "This server is manually installed and controlled from the command line. It might be installed on a remote computer. It must already be running before you can connect to it."

Reply:
@morebrownies for any of the "i" icons that don't show a tip on hover, feel free to provide any copy you think should be in them.

Reply:
Oh sorry wolf, didn't see that Josh kind of did that already.